spyder/widgets/sourcecode/base.py

```diff
@@ -330,30 +332,28 @@ def get_extra_selections(self, key):
         return self.extra_selections_dict.get(key, [])
 
     def set_extra_selections(self, key, extra_selections):
+        # current line has to be highlighted first

```

Comment:
Maybe a docstring would be nice, also stating what key and extra_selections is supposed to be. Like a poors-man typing :-p

Reply:
yep, them I'll improve all docstring of  modified functions (no only the ones that I added)